### DISCUSSION TITLE
Merging multiple samples , comparing GRN's

### DISCUSSION THREAD

**Discussion by brianherb** [aertslab/scenicplus#392] [Q&A]
Hi, Thank you for the great program. I'm interested in merging multiple multiome samples to generate cell-type specific GRN's across multiple samples ( and ideally compare conditions if possible). Looking back over past discussions, there was mention that a multi sample example was here :  https://pycistopic.readthedocs.io/en/latest/Cortex_pycisTopic.html , but that returns a 404 file not found. Does this tutorial exist somewhere else? Thank you! Brian

**Comment by SeppeDeWinter** [comment]
Hi Brian

Thanks for the nice comments.

For multiple samples you can follow the default tutorials (https://pycistopic.readthedocs.io/en/latest/notebooks/human_cerebellum.html, https://scenicplus.readthedocs.io/en/latest/tutorials.html).

Best,

Seppe
